src/linear_algebra/alternating.lean

```diff
@@ -207,6 +207,50 @@ instance [distrib_mul_action Sᵐᵒᵖ N] [is_central_scalar S N] :
 
 end has_smul
 
+/-- The cartesian product of two alternating maps, as a multilinear map. -/
+@[simps { simp_rhs := tt }]
+def prod {N'} [add_comm_monoid N'] [module R N'] (f : alternating_map R M N ι)
```

Comment:
Elsewhere in this file `N'` is used for `add_comm_group`s. Can you add a `P` to the top of the file instead, and give this type `alternating_map R M (N × P)`?